packages/lesswrong/lib/collections/tags/collection.ts

```diff
@@ -15,6 +15,12 @@ const formGroups = {
 };
 
 const schema = {
+  createdAt: {
+    optional: true,
+    type: Date,
+    canRead: ['admins'],
```

Comment:
Same as the other thing

Reply:
Seems fine, but bear in mind these aren't in any fragments and don't show up anywhere

Reply:
Yep, seems fine. I expect we will add this field to some fragments later.

Reply:
Requested changes have been made.

Reply:
(note: the current procedure here is to set the PR's "assigned" status to whoever asked for the changes.

Reply:
(if someone requests changes, that person then has to be the one to approve the PR)

Reply:
+1

Reply:
@Discordius